### ISSUE TITLE
Support for rotate images

### ISSUE THREAD

**Issue by h2non** [DAddYE/vips#11]
Firstly, thank you for this package.

It would be great if you can introduce support for images rotation. It can be done in VIPS. 
Here is the API doc:
http://www.vips.ecs.soton.ac.uk/supported/current/doc/html/libvips/libvips-resample.html#vips-similarity

I think this is the only significant nice-to-have missing feature in the package

**Comment by h2non** [comment]
Rotate implementation from sharp:
https://github.com/lovell/sharp/blob/master/src/resize.cc#L217

**Comment by DAddYE** [comment]
I will add it! Thanks!

**Comment by foresmac** [comment]
+1 on supporting rotation. I had planned to try and add support during our last hack days, but I ended up spending all my time working on my first Django ticket.

**Comment by h2non** [comment]
Due to personal requirements, I've created my own Go package called `bimg`, which is heavily inspired in sharp as well.

All of my proposed features are now supported in `bimg`:
https://github.com/h2non/bimg

**Comment by foresmac** [comment]
We'd certainly entertain the idea of using `bimg` instead, but I'd rather keep this issue open as long as @DAddYE is willing to explore adding support to `vips`.

**Comment by h2non** [comment]
@foresmac I completely agree with you.

**Comment by DAddYE** [comment]
Yes I'm gonna add the support soon, I'm working on trimming which is a feature that I need in another project and I'm also working on improving performances and cpu/memory reduction (1st priority)

**Comment by foresmac** [comment]
Any progress on this issue?
